services/ppac/src/main/resources/application.yaml

```diff
@@ -97,19 +97,19 @@ ppac:
       require-evaluation-type-hardware-backed: ${ANDROID_OTP_REQUIRE_EVALUATION_TYPE_HARDWARE_BACKED:true}
     dat:
       # True to require basicIntegrity for PPAC to pass, false otherwise.
-      require-basic-integrity: ${ANDROID_DAT_REQUIRE_BASIC_INTEGRITY:false}
+      require-basic-integrity: ${ANDROID_DAT_REQUIRE_BASIC_INTEGRITY:true}
       # True to require ctsProfileMatch for PPAC to pass, false otherwise.
-      require-cts-profile-match: ${ANDROID_DAT_REQUIRE_CTS_PROFILE_MATCH:false}
+      require-cts-profile-match: ${ANDROID_DAT_REQUIRE_CTS_PROFILE_MATCH:true}
       # True to require evaluationType to contain BASIC for PPAC to pass, false otherwise.
       require-evaluation-type-basic: ${ANDROID_DAT_REQUIRE_EVALUATION_TYPE_BASIC:false}
       # True to require evaluationType to contain HARDWARE_BACKED for PPAC to pass, false otherwise.
-      require-evaluation-type-hardware-backed: ${ANDROID_DAT_REQUIRE_EVALUATION_TYPE_HARDWARE_BACKED:false}
+      require-evaluation-type-hardware-backed: ${ANDROID_DAT_REQUIRE_EVALUATION_TYPE_HARDWARE_BACKED:true}
     log:
```

Comment:
these should all be false

Reply:
for `/dat` all requirements are turned off now